### PR TITLE
Templating

### PR DESCRIPTION
**Pull request by m4s-b3n**
related to #19, realizing option 2

## FYI
- not tested yet as workflow could not be run dispatched yet
- repo request workflow should now be testable by running dispatched with single line json (parsed issue form body)
- orgs moved to toplevel
- created template directory at toplevel for files required for templates
- used `gh repo create --push` -> not tested yet -> bugfixing after merge ^^

## Questions
- How should the language templates look like?
- How should common templates (`CONTRIBUTING.md`, ....) look like? Do we have templats for that in Xpirit?

### issue form body

```json
{
    "Name": "Bla",
    "Company": "Xpirit",
    "Team": "Team 1",
    "Project": "project 2",
    "Visibility": "internal",
    "Language": "python"
}
```

### json for workflow test
`{"Name":"Bla","Company":"Xpirit","Team":"Team 1","Project":"project 2","Visibility":"internal","Language":"python"}`

### CODE REVIEW

File: .github/workflows/update-repo-request-template.yml [hunk 1]
```diff
@@ -55,20 +63,21 @@ jobs:
...
+      - name: Commit Changes
         if: ${{ steps.change-detection.outputs.changed == 'true' }}
-        env: 
+        env:
+          CI_COMMIT_USER: "xpirit-training-bot[bot]"
+          CI_COMMIT_USER_MAIL: 139118093+xpirit-training-bot[bot]@users.noreply.github.com
           CI_COMMIT_MESSAGE: "chore(org): Update Issue Template File"
-          CI_COMMIT_AUTHOR: Continuous Integration
         run: |
-          git config --global user.name "xpirit-training-bot[bot]"
```
Comment (spindev) [inline .github/workflows/update-repo-request-template.yml:70]: Please put the git user and git email into a repo variable :) same for the other workflow file as well.

File: .github/workflows/repo-request.yml [hunk 2]
```diff
@@ -37,20 +61,43 @@ jobs:
...
+        env:
+          LANGUAGE: ${{ steps.get-attributes.outputs.language }}
+        run: |
+          cp -r ./templates/${LANGUAGE}/* .
+          git add .
+      - name: Create commit
+        env:
+          CI_COMMIT_USER: "xpirit-training-bot[bot]"
+          CI_COMMIT_USER_MAIL: 139118093+xpirit-training-bot[bot]@users.noreply.github.com
+          CI_COMMIT_MESSAGE: "chore(org): Initial commit"
```
Comment (spindev) [inline .github/workflows/repo-request.yml:88]: same here -> into repo variable.

File: .github/workflows/repo-request.yml [hunk 3]
```diff
@@ -1,29 +1,53 @@
-name: Create new repo for repo request 
+name: Create new repo for repo request
 
 on:
- issues:
+  issues:
     types: [labeled]
-    
+  workflow_dispatch:
```
Comment (spindev) [inline .github/workflows/repo-request.yml:6]: nice one 👍 can you add an exmaple json as default? default: ....

**Review by spindev — COMMENTED** [review]
Some comments :)

File: .github/workflows/repo-request.yml (hunk 3 above)
Comment (m4s-b3n) [reply .github/workflows/repo-request.yml:6]: kept it empty for a reason: whenever the issue template changes this default value would have to change as well and you can not check if its done until it fails

would try to automate this in a separate workflow asap

what do you think?

File: .github/workflows/repo-request.yml (hunk 2 above)
Comment (m4s-b3n) [reply .github/workflows/repo-request.yml:88]: mail could be a repo variable, message is different :)

File: .github/workflows/update-repo-request-template.yml (hunk 1 above)
Comment (m4s-b3n) [reply .github/workflows/update-repo-request-template.yml:70]: done

File: .github/workflows/repo-request.yml (hunk 2 above)
Comment (m4s-b3n) [reply .github/workflows/repo-request.yml:86]: done

**Review by spindev — APPROVED** [review]

**Pull request merged**